app/index.html

```diff
@@ -6,6 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <title>crates.io: Rust Package Registry</title>
+    <meta name="description" content="The Rust community’s crate registry" />
+    <meta name="og:title" content="Rust Package Registry" />
+    <meta name="og:description" content="The Rust community’s crate registry" />
+    <meta name="og:image" content="https://crates.io/assets/Cargo-Logo-Small.png" />
```

Comment:
from reading up on the topic it looks like the recommended dimensions of such an image are at least 1200x630, but the linked image is significantly smaller and not really made for this purpose. it might be better to skip the `og:image` until we actually have an image that fits better.